docs/classic.css

```diff
@@ -11,11 +11,12 @@
 
 @import url("basic.css");
 
-/* -- page layout ----------------------------------------------------------- */
+* -- page layout ----------------------------------------------------------- */
 
 body {
-    font-family: Roboto;
-    font-size: 100%;
+    font-family: 'Roboto', 'Arial', 'Helvetica', 'sans-serif';
+/*    font-size: 100%;*/
```

Comment:
Why have the comments?

Reply:
The comments are artifacts of my timid css editing. I guess we can take the training wheels off now, but I am still a bit nervous about burning bridges. My sense is that it's not much freight to carry, but if it bugs you, I could take it out.

Reply:
I think you can just delete them since they are of no use. Because we are using Git, we can always go "back" and see what were the original values in case we need them.